xml/admin_logging.xml

```diff
@@ -0,0 +1,446 @@
+cmd.run "journalctl -u transactional-update"</command>
+ </screen>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.logging.fluentd">
+  <title>External log collection (fluentd)</title>
+  <para>
+   There's a notable exception at infra level, and is that we allow audit logs
+   to be configured from within Velum. This allows the audit logs to be written
+   on the &kube; master nodes at <filename>/var/log/kube-apiserver/audit.log</filename>
+   and an external data collector (continuous reading) like <literal>fluentd</literal>
+   can be used to collect all the audit logs.
+  </para>
+
+  <para>
+   At &kube; level, there are different solutions that can be implemented, for example: <literal>fluentd</literal>
+   can be used to collect all applications log in a central instance (https://docs.fluentd.org/v0.12/articles/kubernetes-fluentd).
+   Then, Elasticsearch and Kibana can be used (https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/)
+   to have an intuitive way to interact with the logs.
+
+   https://docs.fluentd.org/v0.12/articles/kubernetes-fluentd
+  </para>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.logging.log-levels">
+  <title>Log levels</title>
+  <para>
+   There are two main components and their respective sub-components that
+   allow configuration of different loglevels: Salt and &kube;.
+  </para>
+
+  <itemizedlist>
+
+   <listitem>
+    <para>
+     Salt
+    </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        Salt master
+       </para>
+       </listitem>
+       <listitem>
+        <para>
+         Salt minion on each machine
+        </para>
+       </listitem>
+      </itemizedlist>
+    </listitem>
+
+    <listitem>
+     <para>
+      &kube;
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        Master nodes
+       </para>
+       <itemizedlist mark="circle">
+        <listitem>
+         <para>
+          apiserver
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          controller-manager
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          scheduler
+         </para>
+        </listitem>
+       </itemizedlist>
+      </listitem>
+
+      <listitem>
+       <para>
+        All nodes
+       </para>
+       <itemizedlist mark="circle">
+        <listitem>
+         <para>
+          kubelet
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          kube proxy
+         </para>
+        </listitem>
+        </itemizedlist>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+  </itemizedlist>
+
+  <sect2 xml:id="sec.admin.logging.log-levels.salt">
+   <title>Salt Log Levels</title>
+
+   <para>
+    Salt provides different loglevels that apply both to the master and the
+    minions.
+   </para>
+
+   <itemizedlist>
+    <listitem>
+     <para>
+      all
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      garbage
```

Comment:
Maybe we could include a small comment for each loglevel; I think most of them are self-documenting except `garbage`?

Reply:
@ereslibre Where do I find out what these comments should look like? Just in the upstream docs?

Reply:
Well, [salt logging level documentation is **not** that useful](https://docs.saltstack.com/en/latest/ref/configuration/logging/#log-levels); `garbage` level is documented as "Even more debugging information"... Maybe we can check salt internally and provide something a little more useful?

Reply:
`Maybe we can check salt internally and provide something a little more useful?` Uhm, and who can/will do this? I don't have time to dig into Salt source code. If this really makes sense I would add this as a separate task for later.

Reply:
I will have a look later.

Reply:
As mentioned in https://github.com/saltstack/salt/issues/4994#issuecomment-18049810, `garbage` `catches all logging messages, even those not being logged by salt, ie, by external libraries in the current interpreter's scope.`